example/provider-local/gardenadm/shoot.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: garden
 spec:
   region: local
+  secretBindingName: provider-account
```

Comment:
Why do we specify it here now? 👀 We only need it for medium-touch, but this `Shoot` is also used for high-touch. I think this can be confusing for people?

Reply:
Even in the high-touch scenario, the `ControlPlane` might deploy additional components (e.g., cloud-controller-manager) using the `cloudprovider` secret. I think we should support this case.